bookserver/crud.py

```diff
@@ -81,15 +81,16 @@ async def create_answer_table_entry(log_entry: schemas.LogItem):
     values["timestamp"] = datetime.utcnow()
     rslogger.debug(f"hello from create at {values}")
     tbl = answer_tables[EVENT2TABLE[log_entry.event]]
+    # TODO: revalidate now that we know which table to validate against?
     new_entry = tbl(**values)
     async with async_session() as session:
         session.add(new_entry)
+        # TODO: remove this.
```

Comment:
You need the `with async session.begin():` statement if you want to remove the explicit commit.

Reply:
I think I mentioned this before, but I find this mode of operation to be quite awkward.  You modify the code to essentially report a bug.  And I have to respond this way.  So that part of the conversation is in the history of the code but part of it is not.

I would prefer we use discussions or issues for this.

Reply:
I understand. I agree that it's awkward, but it's also a pain to find a problem in my editor, then find it in the github web interface, then file an issue. I'm moving toward this second model, though.

If I'm offline, I can only do the first model.

Reply:
I'll revise my PR to remove it.